sdk/keyvault/test-resources-post.ps1

```diff
@@ -93,7 +93,7 @@ $wrappingFiles = foreach ($i in 0..2) {
 
 # TODO: Use Az module when available; for now, assumes Azure CLI is installed and in $Env:PATH.
 Log "Logging '$username' into the Azure CLI"
-az login --service-principal --tenant $tenant --username $username --password $password
+az login --service-principal --tenant "$tenant" --username "$username" --password="$password"

```

Comment:
Why do we need quotes?

Reply:
Safety. In the case of the password, mine starts with a "-". The quotes alone weren't enough, and even `az` recommended =. But if there are other characters in there that would cause issues, this may help. It certainly doesn't hurt.